initScripts/x86_64/CentOS_7/Docker_17.06.sh

```diff
@@ -439,6 +450,11 @@ main() {
     trap before_exit EXIT
     exec_grp "check_docker_opts"
 
+    if [ ! -z "$SHIPPABLE_HTTP_PROXY" ] || [ ! -z "$SHIPPABLE_HTTPS_PROXY" ] || [ ! -z "$SHIPPABLE_NO_PROXY" ]; then
```

Comment:
This change makes sure even with `install_docker_only` flag, we configure proxy.